azure/table_azure_storage_blob.go

```diff
@@ -462,7 +479,7 @@ func getRowDataForBlob(ctx context.Context, container storage.ListContainerItem,
 			if len(blob.Snapshot) < 1 {
 				isSnapshot = false
 			}
-			items = append(items, blobInfo{blob, blob.Name, account.Name, container.Name, account.ResourceGroup, &subscriptionID, account.Account.Location, isSnapshot})
+			items = append(items, blobInfo{blob, blob.Name, accountName, container.Name, "", &subscriptionID, "", isSnapshot})
```

Comment:
Don't we need to upsert `resourceGroup`?

Reply:
adding resourceGroup at StreamListItem directly. No need to send it in items and iterate from there.